### ISSUE TITLE
Ember test is timing out

### ISSUE THREAD

**Issue by binhums** [twokul/ember-cli-bugsnag#39]
Not sure what's going on here.

**Comment by binhums** [comment]
See comment https://github.com/twokul/ember-cli-bugsnag/pull/33#issuecomment-172639506

**Comment by binhums** [comment]
Getting the following errors running the tests in chrome

![image](https://cloud.githubusercontent.com/assets/831043/13089668/95576b18-d4a6-11e5-84a4-8ec700be8358.png)

```
ember.debug.js:5342 Uncaught Error: Assertion Failed: Ember Views require jQuery between 1.7 and 2.1
ember-testing.js:1159 Uncaught TypeError: $ is not a function
loader.js:110 Uncaught Error: Could not find module `ember` imported from `dummy/app`
index.html:1 Failed to find a valid digest in the 'integrity' attribute for resource 'http://localhost:7357/testem.js' with computed SHA-256 integrity 'lh8NuSGzYSb6XpdfBYsEglnfxBdd6LpuYybvxZC97fM='. The resource has been blocked.
ember.debug.js:5378 DEBUG: For more advanced debugging, install the Ember Inspector from https://chrome.google.com/webstore/detail/ember-inspector/bmdblncegkenkacieihfhpjfppoconhi
```

**Comment by kcin1018** [comment]
@binhums I just looked at the fix_travis branch and it looks like it was the jquery version that is causing the problem.  The ^1.11.3 resolves to an incompatible version of jQuery.  Changing to 1.11.3 fixes these issues.

**Comment by binhums** [comment]
Fixed by #43. Thanks @kcin1018 👍
